ncolony/tests/functional_test.py

```diff
@@ -92,7 +93,7 @@ def main(argv):
     _analyzeLogFile(LOG_FILE)
 
 def _analyzeLogFile(log_file):
-    lines = list(file(log_file))
+    lines = list(open(log_file))
```

Comment:
This will generate a `ResourceWarning`.  A `tox` environment that runs with all warnings is a good thing to have, so you should wrap this and all other unclosed files with `with`.